apps/accounts/api/automations/push_account.py

```diff
@@ -44,7 +44,7 @@ def get_queryset(self):
 
 class PushAccountRecordViewSet(mixins.ListModelMixin, OrgGenericViewSet):
     filterset_class = PushAccountRecordFilterSet
-    search_fields = ('asset__address', 'account_username')
+    search_fields = ('asset__address', 'account__username')
     ordering_fields = ('date_finished',)
     tp = AutomationTypes.push_account
     serializer_classes = {
```

Comment:
I see no irregularities with the given code snippet provided it is complete and correct. No changes are needed.